### ISSUE TITLE
Suggestion: Search list that lets you move to the respective node + shortcut

### ISSUE THREAD

**Issue by elandorr** [juzzlin/Heimer#243]
I just realized I 'wasn't finding' the node that I was certain existed, because the search doesn't actually move you to it, it only gives it a red border.

- that border is near invisible on larger maps
- it's a game of 'find Waldo'
- zooming out 9000 times is annoying
  - Scapple solves this by having a button that zooms out to fit the entire map, and when you let go, drops you off back at the previous zoom level where the mouse is at. That's very practical.
- a list somewhere would be most efficient so you see all occurrences
- but as a quick and easy fix:
  - how about showing a counter [1/9] and jumping to the next on RET?

Maybe I'm just not seeing it, but something like 'CTRL-F' or '/' (user configurable ideally as all things) would be nice, as clicking the little arrow (it hides in smaller windows/screens) and moving the mouse around takes forever.

Thanks for your efforts as usual. I've been restarting Heimer occasionally lately and it's really nice. And open source! No telemetry bullshit to contend with or online-activations, priceless.

**Comment by juzzlin** [comment]
Yeah, I'm aware of this problem and will definitely consider your suggestions. I'm also planning to implement a "magnifying glass" and see how that would work out :)

Also, you can double-click on a node to zoom in.

**Comment by elandorr** [comment]
Thanks!
I can only recommend looking at scapple for ideas. It's not perfect, but feels exceptionally 'natural' compared to everything else in this space. (As natural as a computer can feel...) I went as far as manually cracking it to avoid the online bs, but that takes time and I can't be bothered for every update. (and it's win only) Open source is so much more elegant and I'm thankful for Heimer.

**Comment by sergeevabc** [comment]
@elandorr, just like you, I'm looking forward to get the search improved. Also, the major downside of Scapple is that the developers -- without any coherent rationale -- made their app available only to Windows 10 users, while Heimer works fine on Windows 7.
